pyproject.toml

```diff
@@ -34,7 +34,7 @@ pyproj = '>=2.6'
 requests = '*'
 scipy = '*'
 shapely = '*'
-stormevents = '^1.4.0'
+stormevents = '>=1.4.0'
```

Comment:
I would suggest making this `stormevents = '>=2'` if pre-`2` is causing issues.

Reply:
not aware of issues..